src/RadhydroSimulation.hpp

```diff
@@ -360,6 +362,7 @@ template <typename problem_t> void RadhydroSimulation<problem_t>::readParmParse(
 		amrex::ParmParse hpp("primordial_chem");
 		hpp.query("enabled", enableChemistry_);
 		hpp.query("max_density_allowed", max_density_allowed);
+		hpp.query("min_density_allowed", min_density_allowed);
```

Comment:
maybe add this comment here as well: `// dont do chemistry in cells with densities below the minimum density specified`